packages/teams-js/src/private/files.ts

```diff
@@ -445,58 +457,32 @@ export namespace files {
   }
 
   /**
-   * @hidden
-   * Hide from docs.
-   * ------
-   * Opens a client-friendly preview of the specified file.
+   * @deprecated
+   * As of 2.0.0-beta.6, use only for backwards compatibility of existing code.
    *
-   * @param file - The file to preview.
```

Comment:
Was this something that was added specifically for 2.0? I don't see it anywhere in the updated code which was surprising.

Reply:
This should be added back to privateAPIs.ts

Reply:
This particular API made me realize that we should not actually mark any of these APIs as deprecated. This is because the APIs in the private folder have not had the same  treatment of updating callbacks to return promises for v2. So they don't need to be marked deprecated because there isn't a new standard to use yet. (Meeting and media are exceptions because we did do a pass across all public APIs.)